Makefile

```diff
@@ -130,7 +133,7 @@ ${RESULT_CV_SUMMARY} ${RESULT_TEST_RESULTS} : ${MODEL_ALL} ${RESULT_CV_ALL}
 
 # Report
 
-${FINAL_REPORT_OUTPUT} : {FINAL_REPORT_SOURCE}
-	@{ECHO} "\033[0;37m>> \033[0;33mRendering final report\033[0m"
+${FINAL_REPORT_OUTPUT} : ${RESULT_CV_ALL}
```

Comment:
Should the final_report.Rmd be a source here too, to make the pdf output?

Reply:
Indeed, it would be more complete :+1: